package.json

```
@@ -58,10 +58,14 @@
     "test:evals:llm": "pnpm --filter ./packages/evals test src/metrics/llm",
     "test:observability": "pnpm --filter ./observability/* test",
     "lint-staged": "lint-staged",
+    "lint": "turbo lint --filter \"./packages/**\" --filter \"./stores/**\" --filter \"./auth/**\" --filter \"./voice/**\" --filter \"./deployers/**\" --filter \"./workflows/**\" --filter \"./observability/**\" --filter \"./client-sdks/**\" --filter \"!@mastra/playground\"",
```

You probably want to do this instead
```suggestion
    "lint": "turbo --filter \"!./examples/**/*\" --filter \"!./docs/**/*\" --filter \"!./integrations/**/*\" --filter \"!@mastra/playground\" lint",
```